druntime/mak/COPY

```diff
@@ -243,6 +243,8 @@ COPY=\
 	$(IMPDIR)\core\sys\linux\tipc.d \
 	$(IMPDIR)\core\sys\linux\unistd.d \
 	\
+	$(IMPDIR)\core\sys\linux\linux\if_packet.d \
```

Comment:
stupid question: is this really intended to be `\linux\linux`? why the repeat?

Reply:
It's intentional, because that's what it is in C: "linux/if_packet.h", or the likely full path: "/usr/include/linux/if_packet.h".

Reply:
You just don't get the same repetition in C, because the headers aren't organized to show the system that they go with like we do in druntime.